lib/pull_request.rb

```diff
@@ -125,14 +129,19 @@ def tell_dependency_manager_what_dependencies_are_allowed
   end
 
   def tell_dependency_manager_what_dependabot_is_changing
+    dependency_updates = commit_message.scan(/(?:Bump|Updates) `?(\w+)`?.*from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d+)/)
```

Comment:
Nice, great you were able to consolidate this into one regex! 👏 

It can be simplified slightly (it should only ever be one space):

```suggestion
    dependency_updates = commit_message.scan(/(?:Bump|Updates) `?(\w+)`? from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d+)/)
```